erts/Makefile

```
@@ -153,3 +153,5 @@ release_docs:
 .PHONY: xmllint
 xmllint:
 	$(MAKE) -C doc/src $@
+
+include $(ERL_TOP)/make/app_targets.mk
```

In the top makefile it is probably better to just set the rule explicitly.

```suggestion
test: release release_tests
	$(ERL_TOP)/make/test_target_script.sh $(ERL_TOP)
```

That way we can add targets to the file without them being added to the top makefile

---

Fixed in commit below.